spec/features/course/assessment/submissions_viewing_spec.rb

```diff
@@ -72,23 +72,17 @@
     context 'As a Course Student' do
       let(:user) { create(:course_student, course: course).user }
 
-      scenario 'I can view my submitted and graded submissions' do
```

Comment:
It would work. I didn't want to do that as I would be adding a mess of extra variables to test nearly the same thing.

An array might work, I'll try refactoring the whole file.